app/core.go

```diff
@@ -30,7 +30,7 @@ const (
 	// global-report-channel
 	defaultGlobalReportHour = 15
 	// Gerrit query to use for determining change sets with work needed for the global report
-	defaultWorkNeededQuery = "is:open -is:wip -label:Code-Review=-2 -label:Verified=-1"
+	defaultWorkNeededQuery = "is:open -is:wip -label:Code-Review=-2 -label:Verified=-1 -#onhold -#on-hold"
```

Comment:
I'd like to make this change in our own Storj-specific instance, if it's ok- I've been able to keep Storj-specific code out of this repo for the most part.

Reply:
Would it need to be some extra flag? But, that's completely fine.

Reply:
removed the commit.